Use apprunner ADR

gidsg opened the pull request.

gidsg committed:
Add app runner RFC

samuelhwilliams commented:
I think we can just have this as an ADR rather than a RFC since we're not soliciting commentary for this but - yes :)

gidsg committed:
move to ADR section

As we are recording a decision rather than requesting comments and ADR is more appropriate for this

gidsg commented:
@samuelhwilliams 

Fair point! Moved in 8ef6cb0

samuelhwilliams commented on adr/2025-04-25-app-runner.md:
```diff
@@ -0,0 +1,59 @@
+
+The proposal is not to change and that we will keep the current AWS AppRunner setup until we hit a technical constraint which makes it unviable. 
+
+[ECS Fargage](https://aws.amazon.com/fargate/) offers a range of advanced features not available in AppRunner such as:
```
```suggestion
[ECS Fargate](https://aws.amazon.com/fargate/) offers a range of advanced features not available in AppRunner such as:
```

samuelhwilliams commented on adr/2025-04-25-app-runner.md:
```diff
@@ -0,0 +1,59 @@
+
+We have added an ECS cluster and associated script purely to run ad-hoc tasks such as database migrations. 
+
+We should continue to monitor if AppRunner continues to meet our needs for the main application container.
```
```suggestion
We should continue to monitor if AppRunner meet our needs for the main application container.
```

gidsg committed:
fix typo

Co-authored-by: Samuel Williams <samuelhwilliams@users.noreply.github.com>

gidsg committed:
simplify prose
